### PR TITLE
Lvnodcdxi 30

### PR DESCRIPTION
**Pull request by OstapDutkevych**
fixed the buttons and added a simple logout

### CODE REVIEW

**Review by NadiiaChorna — DISMISSED** [review]

**Review by skiffua — DISMISSED** [review]

File: src/client/components/equalizer/equalizer.js [hunk 1]
```diff
@@ -100,13 +102,45 @@ class Equalizer extends Component {
...
+   }
+
+   pauseSoundFromFile = async () => {
+     const {
+       audioData: {
+         sound,
+         playPauseState,
+       }, playPauseSoundFromFileAsProp,
+     } = this.props;
+     if (playPauseState) {
```
Comment (tonypntsrn) [inline src/client/components/equalizer/equalizer.js:118]: Please, add an extra space after this.props

File: src/client/components/equalizer/equalizer.js [hunk 2]
```diff
@@ -172,9 +206,9 @@ class Equalizer extends Component {
     ctx.fill();
   }
 
-  setCanvasToState = (canvasEl) => {
-    this.setState({
-      ctx: canvasEl.getContext('2d'),
+  setCanvasToState = async (canvasEl) => {
```
Comment (tonypntsrn) [inline src/client/components/equalizer/equalizer.js]: Why this function needed to be asynchronous?

File: src/client/components/logout/logout.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router-dom';
+import { PropTypes } from 'prop-types';
+import { logoutIcon } from '../../assets/icons/icons';
+import './logout.css';
+import Button from '../button';
+
+class Logout extends Component {
+    SignOut = async () => {
```
Comment (tonypntsrn) [inline src/client/components/logout/logout.js]: No need to use a sync here

File: src/client/components/logout/logout.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+      const LogoutButton = (
+        <Button
+          className="ButtonStyleTemplate"
+          icon={logoutIcon}
+          value="Logout"
+          onClick={this.SignOut}
+        />
+      );
+      return (
+        <div>
```
Comment (tonypntsrn) [inline src/client/components/logout/logout.js]: Do you actually need this div here?

Comment (OstapDutkevych) [reply src/client/components/logout/logout.js]: @tonypntsrn 
image
reworked on such an option

Comment (OstapDutkevych) [reply src/client/components/logout/logout.js]: don't need this "div"
reworked for the best option

File: src/client/components/logout/logout.js (hunk 3 above)
Comment (OstapDutkevych) [reply src/client/components/logout/logout.js]: done

File: src/client/components/equalizer/equalizer.js (hunk 2 above)
Comment (OstapDutkevych) [reply src/client/components/equalizer/equalizer.js]: This function should not be asynchronous
fixed it

File: src/client/components/equalizer/equalizer.js (hunk 1 above)
Comment (OstapDutkevych) [reply src/client/components/equalizer/equalizer.js:118]: done

**Pull request closed without merging**